scripts/readme.md

```diff
@@ -92,7 +92,17 @@ Examples:
     Instances of ORJ serving the Field and Register Registers read files containing fields and registers YAML from S3 on startup.
 
     There is an Ansible task in the deployment repo to generate the files in S3 from the application. You will need to run this after reloading the metadata RSF.
-    
+
+### Naming a register with a human-friendly title
+
+The name of a register is immutable because it is its unique ID.  But the
+register can also have a human-readable name, or 'title'.  To set/change it, use
```

Comment:
I would rather call it `register-name` as this is what it is called in the RSF, rather than `title`.